sbt-dotty/sbt-test/scala2-compat/i12109/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.3.12
```

Comment:
That build.properties file isn't needed for a scripted test.

Reply:
I copied everything from the scala-compat test that you added recently.

Reply:
Hmm, I don't see that file in https://github.com/lampepfl/dotty/pull/12120/files, it might have been auto-generated by sbt.